addons/skin.estuary/xml/Includes_MediaMenu.xml

```diff
@@ -123,6 +122,14 @@
 					<include>MediaMenuLabelCommon</include>
 					<label>$LOCALIZE[14018]</label>
 				</control>
+				<control type="button" id="6051">
+					<include>MediaMenuItemsCommon</include>
+					<label>$LOCALIZE[31023]</label>
+					<label2>[B]$INFO[Container.Viewmode][/B]</label2>
+					<visible>Integer.IsGreater(Container.ViewCount,1)</visible>
```

Comment:
Yes, you are right, at least as long as we have the other 'visible' condition in place (which I actually added later). I will remove this one.